TwilioVideo/1.0.0-preview1/TwilioVideo.podspec

```diff
@@ -0,0 +1,19 @@
+Pod::Spec.new do |s|
+  s.name         = "TwilioVideo"
+  s.version      = "1.0.0-preview1"
+  s.summary      = "Twilio Video"
+  s.description  = "The Twilio Video SDK lets you embed real-time voice and video into your iOS apps."
+  s.homepage     = "http://www.twilio.com/video"
+  s.platform     = :ios, "8.1"
+  s.license      = { 
+    :type => "Commercial", 
+    :text => "Copyright 2011-2016 Twilio. All rights reserved. Use of this software is subject to the terms and conditions of the Twilio Terms of Service located at http://www.twilio.com/legal/tos"
+  }
+  s.author       = { "Twilio" => "help@twilio.com" }
+  s.source       = { :http    => "https://media.twiliocdn.com/sdk/ios/video/releases/1.0.0-preview1/twilio-video-ios-1.0.0-preview1.tar.bz2" }
+  s.frameworks            = "AudioToolbox", "VideoToolbox", "AVFoundation", "CoreTelephony", "GLKit", "CoreMedia", "SystemConfiguration"
```

Comment:
The beauties of a dynamic framework. @ptankTwilio Can you update the PR to remove this line?